src/Core/Mesh/MeshPrimitives.inl

```diff
@@ -80,17 +81,17 @@ namespace Ra
                     {
                         Scalar v = Scalar(iv) * Core::Math::PiMul2 / Scalar(V);
 
-                        Core::Vector3 vertex (
+                        Core::Vector3 vertex(
                                 (majorRadius + minorRadius * std::cos(v)) * std::cos(u),
                                 (majorRadius + minorRadius * std::cos(v)) * std::sin(u),
-                                 minorRadius * std::sin(v));
+                                minorRadius * std::sin(v));
 
 
                         result.m_vertices.push_back(vertex);
                         result.m_normals.push_back((vertex - circleCenter).normalized());
 
-                        result.m_triangles.push_back(Triangle(iu*V + iv, ((iu + 1) % U)*V + iv, iu * V + ((iv + 1) % V)));
-                        result.m_triangles.push_back(Triangle(iu * V + iv, iu * V + ((iv + 1) % V), ((iu - 1) % U) * V + ((iv + 1) % V)));
+                        result.m_triangles.push_back(Triangle(iu * V + iv, ((iu + 1) % U) * V + iv, iu * V + ((iv + 1) % V)));
+                        result.m_triangles.push_back(Triangle(((iu + 1) % U) * V + iv, ((iu + 1) % U) * V + ((iv + 1) % V), iu * V + ((iv + 1) % V)));
```

Comment:
This is the relevant part (don't know how tired I was when I wrote the original code...)